src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx

```diff
@@ -0,0 +1,74 @@
+---
+subject: "Docs"
+releaseDate: '2024-06-28'
+version: 'version: June 21-27, 2024'
+---
+
+### New docs
+
+* Mobile monitoring's [application not responding](/docs/mobile-monitoring/mobile-monitoring-ui/application-not-responding) helps you track and analyze when and why your Android application's blocked for more than five seconds.
```

Comment:
> Mobile monitoring's [application not responding]

Feels like there's a noun like `feature` or `condition` that's missing. A possessive on `application not responding` feels weird

Reply:
Good call